### PR TITLE
🌱 Part 16: Scope kubeclient in pkg/virtual/apiexport

### PR DESCRIPTION
**Pull request by varshaprasad96**
## Related issue(s)

Fixes #

### CODE REVIEW

File: pkg/server/config.go [hunk 1]
```diff
@@ -171,12 +172,19 @@ func NewConfig(opts *kcpserveroptions.CompletedOptions) (*Config, error) {
 	c.GenericConfig.RequestInfoResolver = requestinfo.NewFactory() // must be set here early to avoid a crash in the EnableMultiCluster roundtrip wrapper
 
 	// Setup kube * informers
-	c.KubeClusterClient, err = kubernetes.NewClusterForConfig(c.GenericConfig.LoopbackClientConfig)
+	c.KubeClusterClient, err = kubernetes.NewForConfig(kcpclienthelper.NewClusterConfig(c.GenericConfig.LoopbackClientConfig))
 	if err != nil {
 		return nil, err
 	}
+
+	kubeinformerConfig := kcpclienthelper.ConfigWithCluster(c.GenericConfig.LoopbackClientConfig, logicalcluster.Wildcard)
```
Comment (stevekuznetsov) [inline pkg/server/config.go:180]: `kcpclienthelper.ConfigWithCluster` has been removed, please rebase & update these calls

**Comment by stevekuznetsov** [comment]
This needs to be updated after the `kcp-dev/apimachinery` bump in #1780

File: pkg/authorization/delegated/authorizer.go [hunk 2]
```diff
@@ -30,14 +30,14 @@ import (
...
-type DelegatedAuthorizerFactory func(clusterName logicalcluster.Name, client kubeclient.ClusterInterface) (authorizer.Authorizer, error)
+type DelegatedAuthorizerFactory func(clusterName logicalcluster.Name, client kubeclient.Interface) (authorizer.Authorizer, error)
 
 // NewDelegatedAuthorizer returns a new authorizer for use in e.g. admission plugins that delegates
 // to the kube API server via SubjectAccessReview.
-func NewDelegatedAuthorizer(clusterName logicalcluster.Name, client kubeclient.ClusterInterface) (authorizer.Authorizer, error) {
+func NewDelegatedAuthorizer(clusterName logicalcluster.Name, client kubeclient.Interface) (authorizer.Authorizer, error) {
 	delegatingAuthorizerConfig := &authorizerfactory.DelegatingAuthorizerConfig{
 		SubjectAccessReviewClient: &clusterAwareAuthorizationV1Client{
-			AuthorizationV1Interface: client.Cluster(clusterName).AuthorizationV1(),
```
Comment (stevekuznetsov) [inline pkg/authorization/delegated/authorizer.go:40]: Where did this go?

Comment (varshaprasad96) [reply pkg/authorization/delegated/authorizer.go:40]: Passing a unscoped client here: https://github.com/kcp-dev/kcp/pull/1753/files#diff-32c06ebe8bfea0a512b39070c4654c7751769279e2b921a57698df08326f9887R40

**Comment by varshaprasad96** [comment]
Closing this as decision on context based client scoping may not be required.

**Pull request closed without merging**